cfpb/cfgov-refresh
# Update expandable icons to SVG

Spins SVG icon update in privacy act statement secondary navigation expandables from https://github.com/cfpb/cfgov-refresh/pull/3887

## Changes

- Updates expandables in privacy act statement secondary navigation https://github.com/cfpb/cfgov-refresh/pull/3887
- Fixes issue where privacy act statement on /fin-ed/privacy-act-statement/ erroneously had markup that said the expandable was initialized while there was no expandable JS attached to that page.

## Testing

1. Visit /fin-ed/privacy-act-statement/ and resize to mobile size.
2. Inspect the expandable and see that it's an SVG.
3. Do the same for /privacy/

## Screenshots

![screen shot 2018-05-16 at 11 59 42 am](https://user-images.githubusercontent.com/704760/40128821-a620b996-5900-11e8-93ec-3ddb82d6940e.png)
